### PR TITLE
Good handle

### PR DESCRIPTION
**Pull request by yanyixiang**
* [ ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ ] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ ] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

### CODE REVIEW

**Comment by KiloQubit** [comment]
You've opened 3 separate pull requests about this handle, all of them empty.  I'm going to close the other two.  You need to upload your mod to your github branch before opening a pull request.  To add files to this pull request, please upload them to the appropriate directory here: https://github.com/yanyixiang/VoronUsers

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

- Please move your mod into a directory under your username, i.e.

    `printer_mods/yanyixiang/good_handle/`

or

    `printer_mods/YXY/good_handle/`

- Please remove spaces from all file and directory names.
- Please change all filenames and text to English.
- Please add a README.md describing your mod to your directory instead of `螺丝配件.txt`.  In that README, please include all your images, which you can do like

    render 1

- Please add your mod to `printer_mods/README.md` in alphabetical order

**Comment by KiloQubit** [comment]
Thanks for updating the filenames to English!

All the other points I asked for above are still outstanding though.

File: printer_mods/Good handle-Voron/Good handle/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+M5X10(SHCS)               4
+
+M5X12(SHCS)               6
+
+M3X20(SHCS)               6
+
+M3X5X5(HEAT SET INSERT)   6
```
Comment (jtdowney) [inline printer_mods/Good handle-Voron/Good handle/README.md:7]: Is this supposed to the same heatset inserts that are in the BOM? If so, those are M3x5x4.

**Comment by KiloQubit** [comment]
@yanyixiang are you planning to address the outstanding issues raised in the review of this pull request?

**Comment by yanyixiang** [comment]
BOM里的也是可以的，我设计的骡丝孔比较深，建议用加长螺母，这样会更强壮



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "John ***@***.***&gt;; 
发送时间: 2021年8月31日(星期二) 凌晨4:06
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [VoronDesign/VoronUsers] Good handle (#365)





 
@jtdowney commented on this pull request.
 
 
In printer_mods/Good handle-Voron/Good handle/README.md:
 &gt; @@ -0,0 +1,9 @@ +M5X10(SHCS)               4 + +M5X12(SHCS)               6 + +M3X20(SHCS)               6 + +M3X5X5(HEAT SET INSERT)   6  
Is this supposed to the same heatset inserts that are in the BOM? If so, those are M3x5x4.
 
&mdash;
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by KiloQubit** [comment]
Closing due to lack of response from author.

**Pull request closed without merging**